concepts/patterns/introduction.md

```diff
@@ -0,0 +1,75 @@
+# Introduction
+
+Recall from the Fundamentals concept, an AWK program is composed of **pattern-action pairs**.
+
+```awk
+pattern1 { action1 }
+pattern2 { action2 }
+...
+```
+
+## What do we mean by "pattern"?
+
+The "pattern" is any AWK expression that results in a true or false value.
```

Comment:
Are there any expression which cannot be used?

Or, can any expression be used, with no further qualifier needed. Whatever expression is used will be evaluated for truthiness.